src/server.js

```diff
@@ -146,17 +146,17 @@ const initServer = async function (db, config) {
       options: {
         ops: {
           interval: 1000
-        },
-        reporters: {
-          console: [{
-            module: 'good-squeeze',
-            name: 'Squeeze',
-            args: [{ log: '*', response: '*' }]
-          }, {
-            module: 'good-console',
-            args: [{ format: '' }]
-          }, 'stdout']
         }
+        // reporters: {
```

Comment:
If we need to comment this out, we should add a `// TODO: ` with some information about why it's necessary. If not, then we should remove this code block completely.

Reply:
TODO was added